test/e2e/parallel/web_console_plugin_test.go

```diff
@@ -0,0 +1,120 @@
+			Execute(t)
+		response := signupRequest.GetSignupResponse(t)
+
+		// Response should contain a ConsoleURL with a value something like:
+		// https://console-openshift-console.apps.99b682869228f7464338-mgmt.ci.hypershift.devcluster.openshift.com/
+		require.Contains(t, response, "consoleURL")
+
+		// Since we can't easily access the web console API resources directly (due to complex security requirements) we
+		// will instead create a route in the member cluster with which to access the console plugin
+
+		// Before we can create the route however, we need the certificate and private key values from the
+		// member-operator-console-plugin secret
+		secret, err := memberAwait.WaitForSecret(t, "member-operator-console-plugin")
+		require.NoError(t, err)
+
+		// Now we can create the route resource
+		route := &v1.Route{
+			TypeMeta: metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "consolepluginroute",
+				Namespace: memberAwait.Namespace,
+				Annotations: map[string]string{
+					"openshift.io/host.generated": "true",
+				},
+			},
+			Spec: v1.RouteSpec{
+				To: v1.RouteTargetReference{
+					Kind: "Service",
+					Name: "member-operator-console-plugin",
+				},
+				Port: &v1.RoutePort{
+					TargetPort: intstr.FromInt(9443),
+				},
+				TLS: &v1.TLSConfig{
+					Termination: "reencrypt",
+					Certificate: string(secret.Data["tls.crt"]),
+					Key:         string(secret.Data["tls.key"]),
+				},
+				WildcardPolicy: "None",
+			},
+		}
+
+		err = memberAwait.CreateWithCleanup(t, route)
+		require.NoError(t, err)
+
+		reloaded, err := memberAwait.WaitForRouteToBeAvailable(t, route.Namespace, route.Name, "/status")
+		require.NoError(t, err, "route not available", route)
+
+		// Construct the routeURL from the route host
+		routeURL := reloaded.Spec.Host + "/"
+
+		manifestURL := fmt.Sprintf("%s%s%s", "https://", routeURL, "plugin-manifest.json")
+		healthCheckURL := fmt.Sprintf("%s%s%s", "https://", routeURL, "status")
+
+		var tr *http.Transport
+		{ // #nosec G402
+			tr = &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+				},
+			}
+		}
+
+		httpClient := &http.Client{Transport: tr}
+
+		// First perform a health check - if the health check doesn't pass within one minute then the test will fail
+		var healthCheckResponse *http.Response
+
+		req, err := http.NewRequest("GET", healthCheckURL, nil)
+		require.NoError(t, err)
+		req.Header.Set("Authorization", signupRequest.GetToken())
+
+		healthCheckResponse, err = httpClient.Do(req) //nolint
+		require.NoError(t, err)
+		defer healthCheckResponse.Body.Close()
+		require.Equal(t, http.StatusOK, healthCheckResponse.StatusCode, "error polling health check endpoint", route)
+
+		req, err = http.NewRequest("GET", manifestURL, nil)
+		require.NoError(t, err)
+		req.Header.Set("Authorization", signupRequest.GetToken())
+
+		manifestResponse, err := httpClient.Do(req)
+		require.NoError(t, err)
+		defer manifestResponse.Body.Close()
```

Comment:
minor detail - should we check the `manifestResponse.StatusCode` as well ?

Reply:
why not: a02cdbe0